README.md

```diff
@@ -1,10 +1,12 @@
-# discr-game-lab
+# StrategyDSL-lab
 Discrete game Laboratory work system
+This is SPbPU learning project contains:
++ Telegram bot for holding a competition in writing estimators for various discrete antagonistic games with perfect information. 
++ Translator DSL to python
++ Сompetition competition system using alpha-beta pruning
```

Comment:
"Competition" is repeated twice
Alpha-beta pruning is not part of the competition runner: it is the game's implementors responsibility (for now, at least)